web_tool_script_2.R

```diff
@@ -48,20 +48,18 @@ port_col_types <- set_col_types(grouping_variables, "ddddccccddclc")
 ##### EQUITY #####
 ##################
 
+ald_scen_eq <- get_ald_scen("Equity")
+ald_raw_eq <- get_ald_raw("Equity")
```

Comment:
Is this due to the revert from yesterday @maurolepore ?
It seems like a good idea in general to only load this data, if the respective asset class exists, as before. Just as a reminder for ourselves

Reply:
Most likely. Remember I did not author the changes I reverted in #127, so I don't know why the changes were made.

Reply:
I saw the commit that did it in the other PR, will take care of it once this is merged